#1

При запуску py . в консоль появився такий результат:
We are in the __main__
2021-09-30 15:22:44.263465
win32
test

#2

При запуску py . -h в консоль появився такий результат:
usage: . [-h] [-o OPT] [-l]

Приклад передачі аргументів у Python програму.

optional arguments:
  -h, --help            show this help message and exit
  -o OPT, --optional OPT
                        Цей параметр є вибірковим.
  -l, --logs            Якщо виконати команду з цим параметром будуть виводитись логи.


